### PR TITLE
Update Heroku Documentation

### PR DESCRIPTION
**Pull request by dederaditya**
- Fixed missing link from docs
- Updated some images and step by step

### CODE REVIEW

File: docs/GETTING-STARTED.md [hunk 1]
```diff
@@ -71,13 +71,19 @@ You're almost done.
 
 ## Start trading
 
-Log into your Heroku account and open the console:
+Log into your Heroku account and open your deployed app:
 
 ![heroku console](images/image1.png)
 
-Enter `npm run start` into the console and it will start the trading bot, which is now connected to both your Binance and your BVA account.
+To make sure it is running, go to Resources tab, and check if the dyno is switched to on.
```
Comment (dargmuesli) [inline docs/GETTING-STARTED.md:78]:
```suggestion
To make sure it is running, go to Resources tab and check if the dyno is switched to on.
```

File: docs/GETTING-STARTED.md [hunk 2]
```diff
@@ -71,13 +71,19 @@ You're almost done.
...
 
-![heroku run](images/image7.png)
+![heroku run](images/logs.png)
+
+Let's check the bot if is running from logs! You also can check detailed information about the bot from logs.
```
Comment (dargmuesli) [inline docs/GETTING-STARTED.md:82]:
```suggestion
Let's check if the bot is running from the logs. You also can check detailed information about the bot from those logs.
```

File: docs/GETTING-STARTED.md [hunk 3]
```diff
@@ -71,13 +71,19 @@ You're almost done.
...
+
+![heroku_logs](images/running.png)
+
+This is an example if the bot is running! Make sure to switch from All process to Web.
```
Comment (dargmuesli) [inline docs/GETTING-STARTED.md:86]:
```suggestion
This is an example of how it looks if the bot is running. Make sure to switch from All process to Web.
```

**Review by dargmuesli — CHANGES_REQUESTED** [review]

**Review by dargmuesli — APPROVED** [review]

**Comment by dargmuesli** [comment]
Why the draft status?

**Comment by dederaditya** [comment]
Sorry, this is my first time using PR

File: docs/GETTING-STARTED.md [hunk 4]
```diff
@@ -71,13 +71,19 @@ You're almost done.
 
 ## Start trading
 
-Log into your Heroku account and open the console:
+Log into your Heroku account and open your deployed app:
 
 ![heroku console](images/image1.png)
 
-Enter `npm run start` into the console and it will start the trading bot, which is now connected to both your Binance and your BVA account.
```
Comment (dargmuesli) [inline docs/GETTING-STARTED.md:78]: Don't users have to run this command? I'm not using Heroku, so I can't tell.

**Comment by dargmuesli** [comment]
Otherwise, no problem, good PR!

File: docs/GETTING-STARTED.md (hunk 4 above)
Comment (dederaditya) [reply docs/GETTING-STARTED.md:78]: No, it’s not necessary because everytime you build a NodeJS app it will run automatically (from my experience)

Comment (dargmuesli) [reply docs/GETTING-STARTED.md:78]: Alrighty

**Pull request merged**